dot/peerset/handler.go

```diff
@@ -5,15 +5,19 @@ package peerset
 
 import (
 	"context"
+	"sync"
 
 	"github.com/libp2p/go-libp2p-core/peer"
 )
 
 // Handler manages peerSet.
 type Handler struct {
-	actionQueue chan<- action
-	peerSet     *PeerSet
-	closeCh     chan struct{}
+	actionQueue   chan<- action
+	closeCh       chan struct{}
+	writersWG     sync.WaitGroup
+	writerWGMutex sync.Mutex
+
+	peerSet *PeerSet
```

Comment:
nit just putting it back in the original order for less deltas. I'm not sure why it got changed.


```suggestion
	actionQueue   chan<- action
	peerSet     *PeerSet
	closeCh       chan struct{}
	writersWG     sync.WaitGroup
	writerWGMutex sync.Mutex
```